index.html

```diff
@@ -10,38 +10,54 @@
 </head>
 
 <body>
-
+    <section class="main-container">
     <header class="main-header">
```

Comment:
Good job working on your desktop version 😃, but you need to fix some parts 👍 :

- [ ] Kindly, make sure you fixed your `margin` in your `media queries` to replicate the design as much as possible (e.g., font, colors, `images`, text, `margins`) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=ANytl328Q4Wuw0jZ-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.). 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Annotation 2023-03-03 230214](https://user-images.githubusercontent.com/57408419/222829315-c5fb0cc8-c9f8-4ce8-a1af-eeaf959f1030.jpg)| |![Annotation 2023-03-03 23081](https://user-images.githubusercontent.com/57408419/222830188-614e406d-fad5-4bc3-9993-5b58dadd3380.jpg)| |

This is what we are getting when we check the `Responsiveness` of your page

![Annotation 2023-03-03 2308](https://user-images.githubusercontent.com/57408419/222833119-a6266681-9693-4770-bbfc-e03f900233d3.jpg)